var/spack/repos/builtin/packages/xfsprogs/package.py

```diff
@@ -43,7 +44,10 @@ def setup_build_environment(self, env):
         env.append_path("C_INCLUDE_PATH", self.spec["util-linux"].prefix.include.blkid)
 
     def configure_args(self):
-        return ["--with-systemd-unit-dir=" + self.spec["xfsprogs"].prefix.lib.systemd.system]
+        args = ["--with-systemd-unit-dir=" + self.spec["xfsprogs"].prefix.lib.systemd.system]
+        if self.version >= Version("6.11.0"):
```

Comment:
`self.spec.satisfies` avoids a `Version` object from having to be constructed.

Is this also the correct version limit?

Reply:
I tried installing 6.0.0 and it doesn't seem to need this extra argument to build fine, so the change happened somewhere between 6.0.0 and 6.11.0. I can make the change with `self.spec.satisfies`, should I add and try building all the versions in between?